### PR TITLE
feat: krew website updates

### PR DESCRIPTION
**Pull request by nratering**
updated from develop
updated tailwind

### CODE REVIEW

**Comment by mschilling** [comment]
@nratering I've added a space after ':' in the title, so that the 'semantics' check will pass ;-)

**Comment by mschilling** [comment]
In general, it's advised NOT to mix dependencies updates with features/fixes, since it might introduced merge conflicts with (many) other PR's.

**Review by mschilling — APPROVED** [review]
Nice, thanks for the changes! 🚀

File: apps/krew-website/src/pages/index.astro [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Header from '../components/Header.astro';
 import Footer from '../components/Footer.astro';
 
 ---
-<html lang="en">
+<html class="scroll-smooth" lang="en">
```
Comment (guidovanvilsteren) [inline apps/krew-website/src/pages/index.astro:7]: Nothing is done with this class. Or is it already defined somewhere in the CSS?

Comment (nratering) [reply apps/krew-website/src/pages/index.astro:7]: Tailwind class, for smooth scrolling to anchor

**Review by cassshh — APPROVED** [review]

**Review by guidovanvilsteren — APPROVED** [review]

**Pull request merged**